src/stability_sdk/__init__.py

```diff
@@ -0,0 +1,9 @@
+import pathlib, sys
+
+# this is necessary because of how the auto-generated code constructs its imports
+# should be a way to move this upstream
+thisPath = pathlib.Path(__file__).parent.resolve()
```

Comment:
fr I hate this so much. maybe we can generate a python package from api-interfaces and install it properly with a setup.py to get around this jankiness? would greatly prefer it if we had no submodule dependencies, fucks with dependency resolution.

Reply:
Yeah 😢 Would be nice if the SDK was self contained. For a different discussion but perhaps we could have a simple script that grabs api-interfaces (and it's dependencies!) snags the couple needed files for Python and stuffs them in subdirectory. When updating the SDK we just run that script. Users of SDK never have to worry about it or pull down submodules etc.